css-brochure-form/styles.css

```diff
@@ -0,0 +1,49 @@
+.lmore{
+  color:rgb(255, 123, 0);
+  padding-left: 110px;
+  font-family: 'Brandon Grotesque';
+  margin-bottom: 0px;
+  font-style: normal;
+  font-weight: 500;
+}
+div{
+  padding-left: 400PX;
```

Comment:
1. `PX` is not a valid measurement
2. Using padding like this is not the proper way to accomplish this. Are you doing this to try to center the inputs? If so, you should instead set a width on the `form` element, and apply `margin: auto` to center the content with whatever space is left over.